builtin/logical/database/backend.go

```diff
@@ -30,7 +29,7 @@ const (
 
 type dbPluginInstance struct {
 	sync.RWMutex
-	dbplugin.Database
+	database databaseVersionWrapper
```

Comment:
Wouldn't this make `dbPluginInstance` lose all the methods that it inherited from the embedded interface before this change?

Reply:
Ah, we expose equivalent helpers for these methods instead of calling them directly.